cmake/BipedalLocomotionFrameworkFindDependencies.cmake

```diff
@@ -150,6 +150,12 @@ checkandset_dependency(cppad)
 find_package(manif QUIET)
 checkandset_dependency(manif)
 
+find_package(pybind11 CONFIG)
```

Comment:
To be honest I don't remember if ``QUIET`` is needed also here. Is it ``QUIET`` by default?

Reply:
Squashed the change inside a944644